### PR TITLE
Feature admin

### PR DESCRIPTION
**Pull request by nottoast**

### CODE REVIEW

File: src/spoons-tracker-admin/spoons-tracker-admin.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+        <div class="form">
+          <paper-input id="nameField" label="Name" value="{{pubName}}"></paper-input>
+          <paper-input label="Address line 1" value="{{address}}"></paper-input>
+          <paper-input label="City" value="{{city}}"></paper-input>
+          <paper-input label="County" value="{{county}}"></paper-input>
+          <paper-input label="Postcode" value="{{postcode}}"></paper-input>
+          <paper-input label="Latitude" value="{{latitude}}"></paper-input>
+          <paper-input label="Longitude" value="{{longitude}}"></paper-input>
+          <paper-input label="Is it a hotel? (true or false)" value="{{hotel}}"></paper-input>
```
Comment (matkastner) [inline src/spoons-tracker-admin/spoons-tracker-admin.html]: This should be a radio button ? I don't think we can trust Simon to type true or false and it's stored as Boolean in the db

File: src/spoons-tracker-admin/spoons-tracker-admin.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+          <paper-input label="Is it in an airport? (true or false)" value="{{airport}}"></paper-input>
+          <paper-input label="Is it closed? (true or false)" value="{{closed}}"></paper-input>
+        </div>
+
+        <firebase-document id ="editPub"></firebase-document>
```
Comment (matkastner) [inline src/spoons-tracker-admin/spoons-tracker-admin.html]: You could have the value bind here and use pub.name in the fields above, that would cut down on the JavaScript code below and it'd update live.

File: src/spoons-tracker-admin/spoons-tracker-admin.html (hunk 1 above)
Comment (nottoast) [inline src/spoons-tracker-admin/spoons-tracker-admin.html]: haha yeah that's true, I'll update it now

Comment (nottoast) [inline src/spoons-tracker-admin/spoons-tracker-admin.html]: Added and commited

File: src/spoons-tracker-admin/spoons-tracker-admin.html [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+              <template is="dom-repeat" items="{{pubs}}" as="pub" sort="_sortPubs">
+                <paper-item id="{{pub.id}}">[[pub.name]] ([[pub.city]])</paper-item>
+              </template>
+            </paper-listbox>
+          </paper-dropdown-menu>
+        </div>
+
+        <div class="save-button">
+          <paper-item>
+            <span id="buttonLabel">Create</span><paper-icon-button id="save-button" label="save" icon="save" on-tap="_save" suffix></paper-icon-button>
```
Comment (matkastner) [inline src/spoons-tracker-admin/spoons-tracker-admin.html]: The main thing here is the feedback

File: src/spoons-tracker-admin/spoons-tracker-admin.html (hunk 2 above)
Comment (nottoast) [inline src/spoons-tracker-admin/spoons-tracker-admin.html]: I'll look into that mate

File: src/spoons-tracker-admin/spoons-tracker-admin.html (hunk 3 above)
Comment (nottoast) [inline src/spoons-tracker-admin/spoons-tracker-admin.html]: 1. delete button disabled is done, just need to commit.
2. Yeah that sounds good, will add that, using the toast one you mentioned, raised an issue for that
3. Ok will look into that

Comment (nottoast) [inline src/spoons-tracker-admin/spoons-tracker-admin.html]: Added a new issue for paper-card

File: src/spoons-tracker-admin/spoons-tracker-admin.html (hunk 2 above)
Comment (nottoast) [inline src/spoons-tracker-admin/spoons-tracker-admin.html]: Added a new issue for using pub key data bind better on admin form

**Pull request merged**